app/models/registration.rb

```diff
@@ -1,2 +1,7 @@
 class Registration < ApplicationRecord
+  validates :first_name, presence: true
```

Comment:
These validations are super basic, essentially just requiring a value be present for the fields specified. The only one that's a little different is `:gender`, which is requiring that value submitted be included in the given array (male or female).